R/geom_recessions.R

```diff
@@ -1,352 +1,345 @@
-#'Add recessions to time series graphs
```

Comment:
Geom_recessions and geom_text_lastonly files are showing up as completely rewritten in the compare for some reason. Can you briefly note any major changes? I can see the diff correctly in the .Rd help files, so it's really only a question pertaining to the code itself.

Reply:
Ugh, that's annoying. I made the changes on macOS, so the line endings must have been changed from the Windows to Mac/Unix line endings for all the files I edited. Shouldn't impact functionality at all, but does make it harder to review changes. For `geom_recessions`, all I did was simplify the example code. For `geom_text_lastonly`, I added `text_aes` and `point_aes` parameters, behaving exactly as the `text_aes`/`rect_aes` you added to `geom_recessions`. I also tweaked the example code to demonstrate this new functionality.